ros/lib/produce.py

```diff
@@ -1,13 +1,7 @@
 from confluent_kafka import Producer
-from ros.lib.config import INSIGHTS_KAFKA_ADDRESS, GROUP_ID, kafka_auth_config
+from ros.lib.config import kafka_auth_config
 
 
 def init_producer():
-    connection_object = {
-            'group.id': GROUP_ID,
-            'bootstrap.servers': INSIGHTS_KAFKA_ADDRESS,
-            'enable.auto.commit': False
-    }
-    producer = Producer(kafka_auth_config(connection_object))
-
+    producer = Producer(kafka_auth_config())
```

Comment:
Any particular reason for omitting `group.id` and `enable.auto.commit` attributes from here?

Reply:
Okay, I missed your above comment. It makes sense to remove it.